src/Controls/StreetControls.js

```diff
@@ -1,17 +1,41 @@
+import TWEEN from '@tweenjs/tween.js';
 import * as THREE from 'three';
+import { MAIN_LOOP_EVENTS } from 'Core/MainLoop';
 import FirstPersonControls from 'Controls/FirstPersonControls';
-import AnimationPlayer, { AnimatedExpression } from 'Core/AnimationPlayer';
 
-// Expression used to damp camera's moves
-function moveCameraExp(root, progress) {
-    const dampingProgress = 1 - Math.pow((1 - (Math.sin((progress - 0.5) * Math.PI) * 0.5 + 0.5)), 2);
-    root.camera.position.lerpVectors(root.positionFrom, root.positionTo, dampingProgress);
+const material = new THREE.MeshBasicMaterial({
+    side: THREE.DoubleSide,
```

Comment:
Why is double sided?

Reply:
Ok, I fix one surface orientation, so I don't need DoubleSide anymore. Thanks !